apps/studio/src/routes/editor/EditPanel/WindowSettings/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { Button } from '@onlook/ui/button';
```

Comment:
This may be a different PR's work? I think you're just intending to pass the Bypass?

Reply:
Yes, my bad